### ISSUE TITLE
Support `flake8:local-plugins`

### ISSUE THREAD

**Issue by adzenith** [john-hen/Flake8-pyproject#17]
Flake8 supports a [second section](https://flake8.pycqa.org/en/latest/user/configuration.html#using-local-plugins) in the setup.cfg that specifies local plugins. I'm not able to get that to work with this tool, and I took a look at the source and it [looks like](https://github.com/john-hen/Flake8-pyproject/blob/main/flake8p/hook.py#L50) it only checks the normal flake8 section. Any chance support for this other section could be added?
Also just wanted to say thanks and that this project is amazing. 👍

**Comment by john-hen** [comment]
Thanks for pointing this out. Yeah, absolutely, that *can* be added. Do you want to do it?

I guess it's pretty straightforward to implement. Adding a meaningful test will be more difficult. But maybe we can do without a test in this case? I'm open to that if it complicates things to much. (Maybe because I'm won't be using that feature.)

**Comment by christianparpart** [comment]
Hi, I stumbled across this ticket. I just wanted to create a new checker for our own projects and would have created a pylint plugin now rather than a standalone script. I got told that I should seriously consider using flake8 instead and create a flake8 plugin rather a pylint plugin. It seems like this ticket is relevant to me, in order to create flake8 plugin in a git repo and have it self-tested? What do you think? (p.s.: I'm kinda new to how tests are written in Python, so please bare with me) :)

**Comment by john-hen** [comment]
I don't think this issue is relevant to you. This right here *is* a Git repo of a Flake8 plugin that is self-tested.

**Comment by christianparpart** [comment]
@john-hen ok, thx. I'll continue my topic in #23.

**Comment by tmarenko** [comment]
Seems it'd be quite hard. I took a look how flake8 loads plugins and how this project works.
You'd need to pass options for your local plugin [right before plugin loading](https://github.com/PyCQA/flake8/blob/045f297f89a4d6b7f1bb6dc6e62f6eb506aec320/src/flake8/options/parse_args.py#L42) but this project, flake8-pyproject, itself is considered a flake8 plugin. Basically you can't hook anything [before this line](https://github.com/PyCQA/flake8/blob/045f297f89a4d6b7f1bb6dc6e62f6eb506aec320/src/flake8/options/parse_args.py#L51) (before all plugins are registered) so seems it'd be quite difficult to load another plugin additionally after that.

**Comment by tmarenko** [comment]
For anyone interested, I've made tests for `local-plugins` section: https://github.com/john-hen/Flake8-pyproject/commit/5c5a80910226fab9d2a410fd3a45cfd7ff4f3227
